packstream/packstream-specification-1.md

```diff
@@ -775,18 +775,18 @@ utc_nanoseconds = (seconds * 1000000000) + nanoseconds - (tx_offset_minutes * 60
 
 An instant capturing the date, the time, and the time zone.
 
-The time zone information is specified with a **zone identification number**.
+The time zone information is specified with a **zone identifier**.
 
 ```
 DateTime::Structure(
     seconds::Integer,
     nanoseconds::Integer,
-    tz_id::Integer,
```

Comment:
this is the specification with Integer.

Reply:
The [official driver](https://github.com/neo4j/neo4j-java-driver/blob/4.1/driver/src/main/java/org/neo4j/driver/internal/messaging/v2/ValueUnpackerV2.java#L143-L144) uses a String to represent a time zone from the IANA time zone database.

Reply:
this is a bug, it should be `tz_id::String`